app/views/api_docs/reference/reference.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1 class="govuk-heading-xl"><%= t("page_titles.api_docs.reference") %></h1>
+<h1 class='govuk-heading-xl'><%= t('page_titles.api_docs.reference') %></h1>
```

Comment:
Shouldn't it be double quotes for HTML attributes?

Reply:
Aye, always double quotes for HTML attributes.